CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/NoResult/NoResultScreenViewController.swift

```diff
@@ -128,18 +136,11 @@ final public class NoResultScreenViewController: UIViewController {
 
     public override func viewDidLayoutSubviews() {
         super.viewDidLayoutSubviews()
-        var numberOfButtons: CGFloat = 0
-        if viewModel.isEnterManuallyHidden() == false {
-            numberOfButtons += 1
-        }
-        if viewModel.isRetakePressedHidden() == false {
-            numberOfButtons += 1
-        }
         if numberOfButtons > 0 {
             tableView.contentInset = UIEdgeInsets(
                 top: 0,
                 left: 0,
-                bottom: buttonsView.bounds.size.height + numberOfButtons * GiniMargins.margin,
+                bottom: buttonsView.bounds.size.height + CGFloat(numberOfButtons) * GiniMargins.margin,
```

Comment:
[Suggestion] I think the `numberOfButtons` can be declared as CGFloat from the start